comms/dht/src/logging_middleware.rs

```diff
@@ -86,7 +91,7 @@ where
     }
 
     fn call(&mut self, msg: R) -> Self::Future {
-        trace!(target: LOG_TARGET, "{}{}", self.prefix_msg, msg);
+        info!(target: LOG_TARGET, "{}{:?}", self.prefix_msg, msg);
```

Comment:
Perhaps debug?

Reply:
lol, meant debug-level logging (going by the guide of whether a user would be interested in this information)